sections/adguard-specific/annoyances.txt

```diff
@@ -38,6 +38,8 @@ vezess.hu#%#//scriptlet('prevent-setTimeout', 'FbExit', '3000')
 vezess.hu#%#//scriptlet('remove-class', 'blurry', 'body > :not(.m-fbPopup)', 'stay')
 
 origo.hu#$#.oh-holder.open { z-index: 9999 !important; }
+otpbank.hu##.cdk-overlay-container:has(ib-bs-recommended-analytics-consent)
```

Comment:
I think we should wrap this into
```adblock
!#if (!adguard_app_ios && !adguard_ext_android_cb)
otpbank.hu##.cdk-overlay-container:has(ib-bs-recommended-analytics-consent)
otpbank.hu#$#html { position: static !important; }
!#endif
```
because content blockers does not support CSS rules